RFS/src/main/java/org/opensearch/migrations/bulkload/version_os_2_11/GlobalMetadataCreator_OS_2_11.java

```diff
@@ -118,49 +118,41 @@ private List<CreationResult> createTemplates(
             return List.of();
         }
 
-        if (templateAllowlist != null && templateAllowlist.isEmpty()) {
-            log.info("No {} in specified allowlist", templateType);
-            return List.of();
-        }
+        var templatesToCreate = getAllTemplates(templates, templateType);
 
-        var templatesToCreate = getTemplatesToCreate(templates, templateAllowlist, templateType);
-
-        return processTemplateCreation(templatesToCreate, templateType, mode, context);
+        return processTemplateCreation(templatesToCreate, templateType, templateAllowlist, mode, context);
     }
 
-    private Map<String, ObjectNode> getTemplatesToCreate(ObjectNode templates, List<String> templateAllowlist, TemplateTypes templateType) {
+    Map<String, ObjectNode> getAllTemplates(ObjectNode templates, TemplateTypes templateType) {
```

Comment:
Removed the filtering from this method because it prevented reporting templates that were not migrated.